PostHog/PostHogConfig.swift

```diff
@@ -53,6 +53,8 @@ import Foundation
     // only internal
     var disableReachabilityForTesting: Bool = false
     var disableQueueTimerForTesting: Bool = false
+    // internal
+    public var storageManager: PostHogStorageManager?
```

Comment:
I will refactor the access to such global properties later on since this pollutes the public API and people can misuse that even with "internal" comments, but for now, I'd like to quickly fix the bug and that requires access to the storage manager.

Reply:
Maybe worth adding a TODO comment to this effect

Reply:
will make an issue instead.